src/libraries/Microsoft.PowerFx.Core/Parser/ParserOptions.cs

```diff
@@ -37,6 +37,13 @@ public class ParserOptions
         /// It is an immediate parser error if the expression is too long. 
         /// </summary>
         public int MaxExpressionLength { get; set; }
+
+        public ParserOptions(CultureInfo culture = null, bool allowsSideEffects = false, int maxExpressionLength = 0)

```

Comment:
>ParserOptions [](http://example.com/codeflow?start=15&length=13)

nit: adding a ctor (when there were none previously) is a breaking change because it removes the implicit default ctor. 
Consider either:
- not adding hte ctor (treat as a true poco)
- add both a default ctor and additional ctor

Reply:
As all params are optional, we already have a default ctor, no?

Reply:
No, not according to the .net loader. 

A default ctor means no parameters. 
Whereas a default parameter is a compiler-level construct (the compiler will insert the default value at the callsite), but it's still a regular parameter for .net loader.

Reply:
True but I don't think it matters here